Content.Client/Atmos/Monitor/UI/AirAlarmWindow.xaml.cs

```diff
@@ -24,27 +23,27 @@ public sealed partial class AirAlarmWindow : FancyWindow
     public event Action<bool>? AutoModeChanged;
     public event Action? ResyncAllRequested;
 
-    private RichTextLabel _address => CDeviceAddress;
-    private RichTextLabel _deviceTotal => CDeviceTotal;
-    private RichTextLabel _pressure => CPressureLabel;
-    private RichTextLabel _temperature => CTemperatureLabel;
-    private RichTextLabel _alarmState => CStatusLabel;
+    private RichTextLabel Address => CDeviceAddress;
+    private RichTextLabel DeviceTotal => CDeviceTotal;
+    private RichTextLabel Pressure => CPressureLabel;
+    private RichTextLabel Temperature => CTemperatureLabel;
+    private RichTextLabel AlarmState => CStatusLabel;
 
-    private TabContainer _tabContainer => CTabContainer;
-    private BoxContainer _ventDevices => CVentContainer;
-    private BoxContainer _scrubberDevices => CScrubberContainer;
+    private TabContainer TabContainer => CTabContainer;
+    private BoxContainer VentDevices => CVentContainer;
+    private BoxContainer ScrubberDevices => CScrubberContainer;
 
     private Dictionary<string, PumpControl> _pumps = new();
     private Dictionary<string, ScrubberControl> _scrubbers = new();
     private Dictionary<string, SensorInfo> _sensors = new();
-    private Button _resyncDevices => CResyncButton;
+    private Button ResyncDevices => CResyncButton;
 
 
     private Dictionary<Gas, Label> _gasLabels = new();
 
-    private OptionButton _modes => CModeButton;
+    private OptionButton Modes => CModeButton;
 
-    private CheckBox _autoMode => AutoModeCheckBox;
+    private CheckBox AutoMode => AutoModeCheckBox;
```

Comment:
I believe there are a few PR's being reviewed that use these variable names. Unless you are willing to help all those authors fix their PRs, please revert these variable name changes.

Reply:
Got it